server/base/src/main/java/org/apache/accumulo/server/master/balancer/RegexGroupBalancer.java

```diff
@@ -57,9 +57,9 @@ public RegexGroupBalancer(String tableId) {
 
   @Override
   protected long getWaitTime() {
-    Map<String,String> customProps = configuration.getTableConfiguration(tableId).getAllPropertiesWithPrefix(Property.TABLE_ARBITRARY_PROP_PREFIX);
-    if (customProps.containsKey(WAIT_TIME_PROPERTY)) {
-      return ConfigurationTypeHelper.getTimeInMillis(customProps.get(WAIT_TIME_PROPERTY));
+
+    if (tabletBalancerEnvironment.getCustomTableProperties().containsKey(WAIT_TIME_PROPERTY)) {
+      return ConfigurationTypeHelper.getTimeInMillis(tabletBalancerEnvironment.getCustomTableProperty(WAIT_TIME_PROPERTY));
     }
```

Comment:
Hmm I'm not sure how we could abstract the tableConfig into TabletBalancerEnvironment then... only way I can think of is making tableId an optional parameter, only used by *GroupBalancer classes